src/pages/CreatePool/index.tsx

```diff
@@ -0,0 +1,279 @@
+            </Box>
+          </Box>
+
+          <Box mt={4}>
+            <Typography variant="subtitle1">{t("setParameters")}</Typography>
+            <Divider />
+            <Stack direction="row" spacing={3} mt={2}>
+              <Box flex={1}>
+                <Typography mb={2}>{t("setFeeDescription")}</Typography>
+                <TextField
+                  label={`${t("fee")} (%)`}
+                  fullWidth
+                  value={fee}
+                  error={
+                    !digitRegex || parseFloat(fee) > 1 || parseFloat(fee) < 0.04
+                  }
+                  onChange={(e) => setFee(e.target.value)}
+                  helperText={
+                    (!digitRegex ||
+                      parseFloat(fee) > 1 ||
+                      parseFloat(fee) < 0.04) &&
+                    "Fee should be a number between 0.04 and 1"
+                  }
+                />
+              </Box>
+              <Box flex={1}>
+                <Typography mb={2}>{t("amplificationParameter")}</Typography>
+                <Typography mb={2}>
+                  {t("suggestedValue")}
+                  <li>{t("suggestedValue-1")}</li>
+                  <li>{t("suggestedValue-2")}</li>
+                  <li>{t("suggestedValue-3")}</li>
+                </Typography>
+                <TextField
+                  label="A parameter"
+                  onChange={(e) => setParameter(e.target.value)}
+                  error={
+                    parseFloat(parameter) < 100 || parseFloat(parameter) > 400
+                  }
+                  helperText={
+                    (parseFloat(parameter) < 100 ||
+                      parseFloat(parameter) > 400) &&
+                    "Parameter should be a number between 100 and 400"
+                  }
+                  fullWidth
+                />
+              </Box>
+            </Stack>
+          </Box>
+
+          <Stack direction="row" spacing={3} my={4}>
+            <Box flex={1}>
+              <Typography variant="subtitle1">{t("createPoolType")}</Typography>
+              <Divider />
+              <Typography my={2}>{t("createPoolTypeDescription1")}</Typography>
+              <Typography mb={2}>{t("createPoolTypeDescription2")}</Typography>
+              <Typography mb={2}>{t("createPoolTypeDescription3")}</Typography>
+              <ToggleButtonGroup
+                value={poolType}
+                color="secondary"
+                size="large"
+                exclusive
+                onChange={(event, value) => setPoolType(value)}
+                fullWidth
+              >
+                <ToggleButton value="usdMetapool" size="large">
+                  USD Metapool
+                </ToggleButton>
+                <ToggleButton value="btcMetapool">BTC Metapool</ToggleButton>
+                <ToggleButton value="baseMetapool">Base Metapool</ToggleButton>
+              </ToggleButtonGroup>
+            </Box>
+            <Box
```

Comment:
This box should be conditionally rendered based on the selection of `poolType`, that way flexbox will handle resizing.

```
{poolType !== 'basePool' && {<Box> ....
```

Reply:
Currently, `poolType` section handles resizing, but changing speed is high ,so might give user bad user experience.

https://user-images.githubusercontent.com/75422800/159911626-23226ab5-1525-4fb8-aadb-1b36851e5c13.mov

Reply:
Do you want to try adding a transition effect and see how it effects the UX? https://mui.com/components/transitions/

Reply:
That would be awesome!